lib/ansible/plugins/inventory/vultr.py

```diff
@@ -116,8 +116,8 @@ def _retrieve_servers(api_key):
         if not servers_list:
             return []
 
-        return [server for id, server in servers_list.items()]
-    except ValueError as e:
+        return [servers_list[server] for server in servers_list.keys()]
```

Comment:
my previous comment was unclear (s/keys/values/): could not that be `return servers_list.values()` ?